section2/exercises/if_statements.rb

```diff
@@ -63,3 +64,32 @@
 
 cups_of_flour = 1
 has_sauce = true
+if cups_of_flour >= 2 == has_sauce
+  puts "I can make a pizza"
+else
+  puts "I cannot make pizza"
+end
+
+cups_of_flour = 5
+has_sauce = false
+if cups_of_flour >= 2 == has_sauce
+  puts "I can make a pizza"
+else
+  puts "I cannot make pizza"
+end
+
+cups_of_flour = 2
+has_sauce = true
+if cups_of_flour >= 2 == has_sauce
+  puts "I can make a pizza"
+else
+  puts "I cannot make pizza"
+end
+
+cups_of_flour = 3
+has_sauce = true
+if cups_of_flour >= 2 == has_sauce
```

Comment:
Nice solution, it's good to see you tested it with various values in each variable.